app/src/main/resources/ui/configuration-data.js

```diff
@@ -1252,3 +1252,22 @@ dataSourcePropertiesMap.set("slack", {
         }
     }
 });
+
+export let subDataSourceConfigMap = new Map();
+subDataSourceConfigMap.set("http", {
+    method: {
+        displayName: "Method",
+        default: "N/A",
```

Comment:
Maybe best not to use `N/A` as it is confusing. Keeping it as empty as default is best.